document-viewer/src/main/java/org/ebookdroid/ui/viewer/views/BookmarkView.java

```diff
@@ -104,6 +104,9 @@ public void onLongPress(final MotionEvent e) {
             final MotionEvent cancel = MotionEvent.obtain(e);
             cancel.setAction(MotionEvent.ACTION_CANCEL);
             detector.onTouchEvent(cancel);
+			if (cancel != null) {
```

Comment:
I don't think this null check can ever fail, because methods are called on `cancel` earlier